src/Customer.js

```diff
@@ -4,7 +4,7 @@ class Customer extends User {
   constructor(id, name, date, bookings) {
```

Comment:
I do think the order of the constructor parameters compared to the the arguments of the of the super.